docs/root/configuration/access_log.rst

```diff
@@ -228,6 +228,13 @@ The following command operators are supported:
   Local address of the upstream connection. If the address is an IP address it includes both
   address and port.
 
+%UPSTREAM_TRANSPORT_FAILURE_REASON%
+  HTTP
+    The reason of upstream transport failure, this is usually upstream TLS connection failure with 503 response code.
+
+  TCP
+    Not implemented ("-")
```

Comment:
We could implement this, right? Is there a TODO somewhere?

Reply:
Added a TODO, will look into this later to see if I can implement within this PR.